posthog/hogql/test/test_expr_parser.py

```diff
@@ -0,0 +1,164 @@
+from typing import Any, Dict
+
+from posthog.hogql.expr_parser import ExprParserContext, translate_hql
+from posthog.test.base import APIBaseTest, ClickhouseTestMixin
+
+
+class TestExprParser(APIBaseTest, ClickhouseTestMixin):
```

Comment:
Note: I think these tests would be simpler to write and maintain using `@pytest.mark.parametrize(`

Reply:
For sure. Next time? 🤞 😆

Reply:
Yup - non-blocking!